include/ensmallen_bits/cmaes/cmaes_impl.hpp

```diff
@@ -110,11 +110,13 @@ typename MatType::elem_type CMAES<SelectionPolicyType>::Optimize(
   for (size_t f = 0; f < numFunctions; f += batchSize)
   {
     const size_t effectiveBatchSize = std::min(batchSize, numFunctions - f);
-    const ElemType objective = function.Evaluate(mPosition[0], f,
+    const ElemType objective = 
+        function.Evaluate(transformationPolicy.Transform(mPosition[0]), f,
         effectiveBatchSize);
     currentObjective += objective;
 
-    Callback::Evaluate(*this, function, mPosition[0], objective,
+    Callback::Evaluate(*this, function, 
+      transformationPolicy.Transform(mPosition[0]), objective,
```

Comment:
```suggestion
        transformationPolicy.Transform(mPosition[0]), objective,
```
Minor style fix: wrapped lines should be doubly indented.

Reply:
Also worth pointing out is that you call `transformationPolicy.Transform()` twice here for each inner iteration of the loop.  Does it perhaps make more sense to call `Transform()` once, before the loop?  That may require some further refactoring, but it seems to me to be conceptually simpler to simply use `Transform()` to map each new proposed iterate back into the required range right after a step is taken, only once.

Reply:
Ahh yes! Sorry and thank you for pointing this out! I will fix this very soon.

Reply:
> That may require some further refactoring, but it seems to me to be conceptually simpler to simply use `Transform()` to map each new proposed iterate back into the required range right after a step is taken, only once.

This would make the update steps depend on the transformation policy, which isn't really my intention here.